### ISSUE TITLE
CoNSeP embedding analysis code

### ISSUE THREAD

**Issue by EmanuelSoda** [TIO-IKIM/CellViT#28]
Hi, first of all, thanks for this amazing tool! 

I am training a CellViT instance on another dataset and I would like to perform a cell embedding analysis as you have done in your manuscript with CoNSeP. Unfortunately, looking at the provided code in the repo, I wasn't able to find the relative code. Since I am fairly new to this field could you please provide the CoNSeP analysis so that I'll try to adapt my dataset?

Thanks in advance

**Comment by florianbuettner** [comment]
I agree with Emanuel, it would be helpful if you could briefly show how you assign the embeddings to the predicted instances.

**Comment by 111abc-rt** [comment]
Could you please provide the complete code for training, testing, and inference on the CoNSeP dataset for segmentation tasks? We would appreciate it if you could supply code for both scenarios: with nucleus classification and without nucleus classification (binary segmentation).
